Auth0/Auth0Authentication.swift

```diff
@@ -360,6 +347,20 @@ private extension Auth0Authentication {
                        telemetry: self.telemetry)
     }
 
+    func tokenExchange() -> Request<Credentials, AuthenticationError> {
```

Comment:
Not sure if you want to change these, but strictly speaking there's no "Token Exchange" going on with these defaults, you're just calling the "Get Token" endpoint https://auth0.com/docs/api/authentication#dynamic-application-client-registration

Reply:
Renamed the private method to just `token()` after the endpoint in https://github.com/auth0/Auth0.swift/pull/573/commits/8f5a6561e3739ce4f122668f5ef2135481789ce4.